### ISSUE TITLE
"Unsupported browser" state of login page doesn't explain what's going on.

### ISSUE THREAD

**Issue by mvollmer** [cockpit-project/cockpit#14884]
When Cockpit detects a missing feature, I get this:

![image](https://user-images.githubusercontent.com/3228183/98520963-7ab5f900-227b-11eb-915f-087958b12904.png)

It just says "Download a new browser", but not why.

**Comment by garrett** [comment]
It's probably rewriting the header?

There's an H2 that mentions the browser is out of date, and it should be showing up in this case.

https://github.com/cockpit-project/cockpit/blob/master/src/ws/login.html#L109:
```html
          <h2 class="unsupported-browser-heading" translate>A modern browser is required for security, reliability, and performance.</h2>
```

**Comment by mvollmer** [comment]
The header seems to be moved off-screen explicitly by the CSS:

```
  .unsupported-browser-heading {
    font-size: 3rem;
    left: 50%;
    /* Pull the H2 half the width to the left to center */
    margin: 0 0 0 -32rem;
    position: absolute;
    text-align: center;
    top: -16rem;
    width: 64rem;
  }
```

**Comment by garrett** [comment]
Pretty sure it's not supposed to be explicitly moved off screen, but should be at the center:

```
    /* Pull the H2 half the width to the left to center */
```

**Comment by garrett** [comment]
The message was originally supposed to look like the screenshots in:
https://github.com/cockpit-project/cockpit/issues/9438

This is a screenshot of what it wound up looking like, I think?

![log in screenshot - unsupported](https://user-images.githubusercontent.com/10246/43536144-276afbe0-95bc-11e8-99bf-ead435795e0c.png)

Things have obviously changed since then. 

Meanwhile, Chromium-based Edge supports Windows 7+ and can be downloaded and is "bundled" in new versions of Windows 10. So the messaging isn't quite right. Safari is still bundled, however.

Anyway, we probably want to rethink this screen and improve JS support too.

But, meanwhile, we can address this specific issue by fixing the visibility of the heading.

**Comment by garrett** [comment]
This will be fixed by PR #14891.
